models/intel/human-pose-estimation-0001/README.md

```diff
@@ -41,5 +41,14 @@ The net outputs are two blobs:
 1. Name: `Mconv7_stage2_L1`, shape: `1, 38, 32, 57` contains keypoint pairwise relations (part affinity fields).
 2. Name: `Mconv7_stage2_L2`, shape: `1, 19, 32, 57` contains keypoint heatmaps.
 
+## Demo usage
+
+The model can be used in the following demos provided by the OpenVINO toolkit to show its capabilities:
+
+* [Human Pose Estimation C++ Demo](../../../demos/human_pose_estimation_demo/cpp/README.md)
+* [Human Pose Estimation Python\* Demo](../../../demos/human_pose_estimation_demo/python/README.md)
+* [Multi-Channel Human Pose Estimation C++ Demo](../../../demos/multi_channel_human_pose_estimation_demo/cpp/README.md)
+* [Single Human Pose Estimation Demo](../../../demos/single_human_pose_estimation_demo/python/README.md)
```

Comment:
I do not see this model in supported list for single human pose estimation 
https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/single_human_pose_estimation_demo/python
https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/single_human_pose_estimation_demo/python/models.lst

Reply:
Fixed